pkg/client/cli/intercept/command.go

```diff
@@ -105,6 +110,20 @@ func (c *Command) AddInterceptFlags(cmd *cobra.Command) {
 		flagSet.Lookup("replace").Deprecated = "Use the replace command."
 	}
 
+	// HTTP Intercepts flags
+	flagSet.StringSliceVar(&c.HeaderFilters, "header", nil,
```

Comment:
Maybe also permit colon as a separator? That way, the flag value can also be used for the curl `-H` flag.